ui/apps/platform/src/Containers/Integrations/IntegrationsListPage/IntegrationsTable.tsx

```diff
@@ -222,7 +223,9 @@ function IntegrationsTable({
                                         <Td
                                             actions={{
                                                 items: actionItems,
-                                                disable: !permissions[source].write,
+                                                disable:
+                                                    !permissions[source].write ||
+                                                    !isUserResource(integration.traits),
```

Comment:
There is a subtle counterpart that first cell of row has checkbox for multiple delete action.

```ts
                                {hasMultipleDelete && (
                                    <Th
                                        select={{
                                            onSelect: onSelectAll,
                                            isSelected: allRowsSelected,
                                        }}
                                    />
                                )}
```

Unless you see an easy solution, I recommend to close that loophole as a follow up. Here is docs page:
https://www.patternfly.org/v4/components/table

Reply:
Good catch!  Created a follow up ticket: https://issues.redhat.com/browse/ROX-17313